modules/experimental/airgap_down.sh

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env bash
+
+#!/usr/bin/env bash
+
+. ./defaults.sh
+. ../../include/common.sh
+. .envrc
+
+if [[ ${BACKEND} != "caasp4os" ]]; then
```

Comment:
I would like to use the cap-values `platform` key better, that way it will work if people don't call it with the BACKEND var. See https://github.com/SUSE/catapult/blob/db7da28355dfb2d7034de67bbc50aff36ef396a6/backend/caasp4os/deploy.sh#L120

Reply:
`BACKEND` always needs to be set for catapult. That line you linked requires `caasp4os` to be the `BACKEND` in order to generate that configmap. Users don't need to set backend every time they run catapult make targets however, the backend info is in the .envrc  file

Reply:
Checking `BACKEND` is more reliable than checking the configmap as well, as you may need to run certain targets when the kubernetes cluster may be unavailable, like kubeconfig, charts, gen-config, cleaning, etc.